staging/src/k8s.io/apiserver/pkg/server/options/recommended.go

```diff
@@ -57,9 +62,20 @@ func (o *RecommendedOptions) AddFlags(fs *pflag.FlagSet) {
 	o.Audit.AddFlags(fs)
 	o.Features.AddFlags(fs)
 	o.CoreAPI.AddFlags(fs)
+	o.Admission.AddFlags(fs)
 }
 
-func (o *RecommendedOptions) ApplyTo(config *server.RecommendedConfig) error {
+// ApplyTo adds RecommendedOptions to the server configuration.
+// scheme is the scheme of the apiserver types that are sent to the admission chain.
+// pluginInitializers can be empty, it is only need for additional initializers.
+func (o *RecommendedOptions) ApplyTo(
+	config *server.RecommendedConfig,
+	scheme *runtime.Scheme,
+	pluginInitializers []admission.PluginInitializer,
+) error {
+	if scheme == nil {
```

Comment:
we could move this into the `o.Admission != nil` branch. I think that's what @caesarxuchao proposes. I would like that as well actually.

Reply:
Sounds resonable